swift/WorkflowTesting/Tests/WorkflowRenderTesterTests.swift

```diff
@@ -121,7 +168,6 @@ private struct TestWorkflow: Workflow {
     }
 
     func workflowDidChange(from previousWorkflow: TestWorkflow, state: inout TestWorkflow.State) {
-        // TODO: Add a behavior when it changes to validate.
```

Comment:
Testing `workflowDidChange` can (and should) be done directly against a workflow, not with the render tester - so this comment doesn't make sense.